$OpenBSD: patch-makefile,v 1.2 2021/08/23 05:38:49 rsadowski Exp $

- First hunk: fix directories
- Second hunk: fix `prosody.version` target to stop reporting prosody
  version as `unknown`

Index: makefile
--- makefile.orig
+++ makefile
@@ -1,12 +1,12 @@
 
 include config.unix
 
-BIN = $(DESTDIR)$(PREFIX)/bin
+BIN = $(DESTDIR)$(PREFIX)/sbin
 CONFIG = $(DESTDIR)$(SYSCONFDIR)
 MODULES = $(DESTDIR)$(LIBDIR)/prosody/modules
 SOURCE = $(DESTDIR)$(LIBDIR)/prosody
 DATA = $(DESTDIR)$(DATADIR)
-MAN = $(DESTDIR)$(PREFIX)/share/man
+MAN = $(DESTDIR)$(PREFIX)/man
 
 INSTALLEDSOURCE = $(LIBDIR)/prosody
 INSTALLEDCONFIG = $(SYSCONFDIR)
@@ -90,12 +90,12 @@ prosody.cfg.lua.install: prosody.cfg.lua.dist
 	sed 's|certs/|$(INSTALLEDCONFIG)/certs/|' prosody.cfg.lua.dist > $@
 
 prosody.version:
-	test -f prosody.release && \
-		cp prosody.release $@ || \
-		test -f .hg_archival.txt && \
-		sed -n 's/^node: \(............\).*/\1/p' .hg_archival.txt > $@ || \
-		test -f .hg/dirstate && \
-		hexdump -n6 -e'6/1 "%02x"' .hg/dirstate > $@ || \
-		echo unknown > $@
-
-
+	if [ -f prosody.release ]; then \
+		cp prosody.release $@; \
+	elif [ -f .hg_archival.txt ]; then \
+		sed -n 's/^node: \(............\).*/\1/p' .hg_archival.txt > $@; \
+	elif [ -f .hg/dirstate ]; then \
+		hexdump -n6 -e'6/1 "%02x"' .hg/dirstate > $@; \
+	else \
+		echo unknown > $@; \
+	fi
